shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/resultset/DatabaseMetaDataResultSet.java

```diff
@@ -358,10 +350,27 @@ public Object getObject(final String columnLabel) throws SQLException {
     @Override
     public int findColumn(final String columnLabel) throws SQLException {
         checkClosed();
-        if (!columnLabelIndexMap.containsKey(columnLabel)) {
-            throw new SQLException(String.format("Can not find columnLabel %s", columnLabel));
+
+        Integer columnIndex = columnLabelIndexMap.get(columnLabel);
```

Comment:
@rfscouto The implementation of this logic is a bit complicated. Can we consider using `Map<String, Integer> columnLabelIndexMap  = new TreeMap<>(String.CASE_INSENSITIVE_ORDER)` to solve this exception?

Reply:
That is completely fine by me. It is exactly what I had initial though about, but then read https://github.com/apache/shardingsphere/issues/5432 and thought we wanted to be aligned with the implementation of one of the big ones (and I made it aligned with PostgreSQL).
Converted to TreeMap.